diff -Naur --exclude=.svn --exclude=RELEASE /home/oxygen/MOONEY/junk/synApps_5_8/support/ipac-2-13/drvIpac/drvIpac.dbd ipac-2-13/drvIpac/drvIpac.dbd
--- /home/oxygen/MOONEY/junk/synApps_5_8/support/ipac-2-13/drvIpac/drvIpac.dbd	2013-07-26 15:21:38.212245000 -0500
+++ ipac-2-13/drvIpac/drvIpac.dbd	2014-12-16 12:59:18.139136000 -0600
@@ -5,11 +5,11 @@
 
 # Register your carrier driver(s) with these
 #registrar(mv162ipRegistrar)
-#registrar(vipc310Registrar)
-#registrar(vipc610Registrar)
-#registrar(vipc616Registrar)
-#registrar(tvme200Registrar)
-#registrar(xy9660Registrar)
+registrar(vipc310Registrar)
+registrar(vipc610Registrar)
+registrar(vipc616Registrar)
+registrar(tvme200Registrar)
+registrar(xy9660Registrar)
 #registrar(Hy8002Registrar)
 
 # The ATC40 carrier builds on ISA-bus (x86) systems only:
